assets/js/order-script.js

```diff
@@ -154,4 +154,17 @@ jQuery( function( $ ) {
 		}
 	}
 
+	jQuery('#wpo_wcpdf-data-input-box').on('click', '.view-more', function(e) {
+		e.preventDefault();
+		jQuery(this).addClass('pdf-hide');
```

Comment:
You can use the [`show`](http://api.jquery.com/show/) & [`hide`](http://api.jquery.com/hide/) functions instead of a class.